packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -48,7 +49,8 @@ export default {
             isAsapAvailable: false
         },
         authToken: '',
-        isLoggedIn: false
+        isLoggedIn: false,
+        geolocation: null
```

Comment:
Nit: in line 225 we're camel-casing `geoLocation`, so we should camel-case here it as well.

Reply:
I did have camel casing before but if I use this casing then it matches the casing on the outgoing model in the Submit, so then I can replace like for like in the mapper rather than assigning one to the other.